addons/calendar/tests/test_attendees.py

```diff
@@ -111,3 +111,8 @@ def test_default_attendee(self):
         })
         self.assertIn(self.partner, event.attendee_ids.partner_id, "Partner should be in attendee")
         self.assertNotIn(partner_bis, event.attendee_ids.partner_id, "Partner bis should not be in attendee")
+
+    def test_write_attendee_ids(self):
```

Comment:
Hmm, actually, after second reading, it seems that there are "test_attendee_added" and "test_attendee_added_create", so this should be renamed to "test_attendee_added_write" and re-ordered at the correct spot.

As we are doing the same thing but using the write method instead of assigning/creating.

Does that make sense?

Reply:
Here I used 'adding' as an example but since it's not specific to adding or removing I thought I'd keep it apart. Either works though, just need to make sure someone doesn't come around and replace the write with an `=` or something. Docstring should be enough for that.

Reply:
I think you were a bit hasty and called it "test_write_attendee_added_write" instead of "test_attendee_added_write" ^^